python/test.py

```diff
@@ -0,0 +1,25 @@
+import re
```

Comment:
For testing purpose, you should use a unittest framework, such as "https://docs.python.org/fr/3/library/unittest.html"

Reply:
Good idea thanks, I didn't know this framework